pkg/utils/time.go

```diff
@@ -0,0 +1,127 @@
+package utils
+
+/*
+#include <unistd.h>
+*/
+import "C"
+
+import (
+	"os"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"golang.org/x/sys/unix"
+
+	"github.com/aquasecurity/tracee/pkg/logger"
+)
+
+var configHZOnce, clockTickOnce sync.Once
+var configHZ int
+var userHZ int64
+
+// GetSystemHZ returns an approximation of CONFIG_HZ (the kernel timer interrupt).
+func GetSystemHZ() int {
+	configHZOnce.Do(
+		func() {
+			jiffiesStart := getBootTimeInJiffies()
+			time.Sleep(time.Second)
+			jiffiesEnd := getBootTimeInJiffies()
+			inferredHz := jiffiesEnd - jiffiesStart
+			configHZ = RoundToClosestN(int(inferredHz), 50) // round to closest 50Hz (100, 150,...)
+		},
+	)
+	return configHZ // CONFIG_HZ
+}
+
+// GetUserHZ returns USER_HZ (the user-space timer interrupt), the system clock tick rate.
+func GetUserHZ() int64 {
+	// USER_HZ is 100HZ in almost all cases (untrue for embedded and custom builds).
+	clockTickOnce.Do(
+		func() {
+			userHZ = int64(C.sysconf(C._SC_CLK_TCK))
+		},
+	)
+	return userHZ // USER_HZ
+}
+
+func getBootTimeInJiffies() int64 {
+	data, err := os.ReadFile("/proc/timer_list")
+	if err != nil {
+		logger.Debugw("error reading /proc/timer_list", "err", err)
+		return 0
+	}
+	lines := strings.Split(string(data), "\n")
+	for _, line := range lines {
+		if strings.HasPrefix(line, "jiffies:") {
+			fields := strings.Fields(line)
+			if len(fields) >= 2 {
+				bootTimeInJiffies, err := strconv.ParseInt(fields[1], 10, 64)
+				if err != nil {
+					logger.Debugw("error parsing jiffies value", "err", err)
+					continue
+				}
+				return bootTimeInJiffies
+			}
+		}
+	}
+	logger.Debugw("jiffies value not found")
+
+	return 0
+}
+
+//
+// Boot time functions
+//
+
+// GetStartTimeNS returns the system start time in nanoseconds (using CLOCK_MONOTONIC).
+func GetStartTimeNS() int64 {
+	var ts unix.Timespec
+	err := unix.ClockGettime(unix.CLOCK_MONOTONIC, &ts)
+	if err != nil {
+		logger.Debugw("error getting monotonic time", "err", err)
+		return 0
+	}
+	return ts.Nano()
+}
+
+// GetBootTimeNS returns the boot time of the system in nanoseconds.
+func GetBootTimeNS() int64 {
```

Comment:
Even though I originally wrote this code, it was hard to me to understand it now (This is NOT the real boot time, just a helper for tracee).
Then I remembered that I had some comments in the original code, which were now removed:
```
	// Tracee bpf code uses monotonic clock as event timestamp. Get current
	// monotonic clock so tracee can calculate event timestamps relative to it.

	var ts unix.Timespec
	err = unix.ClockGettime(unix.CLOCK_MONOTONIC, &ts)
	if err != nil {
		return errfmt.Errorf("getting clock time %v", err)
	}
	startTime := ts.Nano()

	// Calculate the boot time using the monotonic time (since this is the clock
	// we're using as a timestamp) Note: this is NOT the real boot time, as the
	// monotonic clock doesn't take into account system sleeps.
```

Can you please add those comments back?